### PR TITLE
feat: add getBalance method 

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: console/src/main/java/org/nervos/ckb/TransactionClient.java [hunk 1]
```diff
@@ -26,11 +28,18 @@ public static void main(String[] args) throws Exception {
                 "ckt1qyq9ngn77wagfurp29738apv738dqgrpqpssfhr0l6", new BigInteger("12000000000")),
             new Receiver(
                 "ckt1qyq2pmuxkr0xwx8kp3ya2juryrygf27dregs44skek", new BigInteger("15000000000")));
+    System.out.println("Sender's balance: " + getBalance(senderPrivateKey).toString(10));
```
Comment (ashchan) [inline console/src/main/java/org/nervos/ckb/TransactionClient.java]: We should consider remove this `console` package. The example here is already simple enough to just appear on doc.

File: core/src/main/java/org/nervos/ckb/transaction/CellGatherer.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            break;
+          }
+        }
+      }
+      fromBlockNumber = currentToBlockNumber + 1;
+    }
+    return new Cells(cellInputs, new BigDecimal(inputsCapacities).toBigInteger());
+  }
+
+  public BigInteger getCapacitiesWithPrivateKey(String privateKey) throws IOException {
```
Comment (classicalliu) [inline core/src/main/java/org/nervos/ckb/transaction/CellGatherer.java]: Using `privateKey` to getBalance seems strange.
I think using `address/lockHash/publicKey` is enough.

Comment (duanyytop) [reply core/src/main/java/org/nervos/ckb/transaction/CellGatherer.java]: Fixed https://github.com/nervosnetwork/ckb-sdk-java/pull/152/commits/3aee99614bdd9915dbebb2191090c0c494cb552c

File: console/src/main/java/org/nervos/ckb/TransactionClient.java (hunk 1 above)
Comment (duanyytop) [reply console/src/main/java/org/nervos/ckb/TransactionClient.java]: `console` package can generate all modules to one jar file, not just an example package. 
So I suggest we can rename `TransactionClient` to `TransactionExample` .

**Pull request merged**